release notes/v0.34.0.md

```diff
@@ -0,0 +1,59 @@
+k6 v0.34.0 is here! :tada: It introduces the already announced `k6/execution` API and includes some more enhancements and a bunch of minor bug fixes.
+
+## New k6 JavaScript API - `k6/execution`
+
+A new JS module with the `k6/execution` import was introduced. It allows the fetching of information about the current VU, instance (mostly relevant in distributed/cloud) or scenario execution state through a series of exported properties.
```

Comment:
Given the changed heading I am for dropping of the whole first sentence
```suggestion
The `k6/execution` module allows the fetching of information about the current VU, instance (mostly relevant in distributed/cloud) or scenario execution state through a series of exported properties.
```
Maybe even `k6/execution` should link directly to it's JS API docs

Reply:
maintain it as unresolved so I will add the link when the doc will be merged